### ISSUE TITLE
Some more questions

### ISSUE THREAD

**Issue by albertma-evotec** [insilicomedicine/GENTRL#17]
in gentrl.py, the parameter cond_lb and cond_rb was not used in the body? What is the purpose of them? 
and why there are two learning-rate variable (lr_lp and lr_dec)? is lr_dec referring to the decay rate?
 
Thanks
```
def train_as_rl(self,
                    reward_fn,
                    num_iterations=100000, verbose_step=50,
                    batch_size=200,
                    cond_lb=-2, cond_rb=0,
                    lr_lp=1e-5, lr_dec=1e-6):
```

**Comment by Bibyutatsu** [comment]
1. Yeah, you are right `cond_lb` and `cond_rb` are not used. Still don't know what is the purpose.

2. `lr_lp` refers to the **learning rate of Learnable Prior** and `lr_dec` refers to the **learning rate of Decoder**

As in the reinforcement learning step, only the `lp` and `dec` are trained so `optimizer_lp` and `optimizer_dec` are used to update the parameters
